devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -23,6 +23,8 @@ metadata {
         capability "Health Check"
         capability "Sensor"
         capability "Configuration"
+        
+        command "reset"

```

Comment:
As @PKacprowiczS mentioned before, custom commands are not supported anymore in new ST application.
Please remove that line.
You might call reset() from configure() method, at the end, for example: 
` ...
zigbee.simpleMeteringPowerConfig() + reset()
`